src/components/CommunityProjectsCard/CommunityProjectsStyles.module.css

```diff
@@ -0,0 +1,26 @@
+.singleCardWrapper {
```

Comment:
Whoa, I had no idea this was a thingy in CRA!

Reply:
Yeahhh its pretty cool. johnhok shared it with me.